app/pages/browse-solutions/template.njk

```diff
@@ -20,7 +20,7 @@
     <div class="nhsuk-grid-row nhsuk-promo-group" data-test-id="browse-solutions-promo-group">
       <div class="nhsuk-grid-column-one-half nhsuk-promo-group__item nhsuk-u-padding-left-0" data-test-id="all-solutions-promo">
         {{ promo({
-          "href": "/solutions/all",
+          "href": "/solutions/capability-selector",
```

Comment:
I know this may have been partly my fault but can we stick with `capabilities`

Reply:
if by partly you mean all then yeah 😄